src/tasks/deploy.mk

```diff
@@ -60,10 +60,10 @@ deploy-canary: ## deploy-canary: deploy canary app to staging
 
 	$(MAKE) change-api
 
-deplo%: ## deploy: deploy the app to heroku
+deploy-staging: ## deploy-staging: deploy the app to staging.
 	$(call ASSERT_VARS_EXIST, HEROKU_APP_STAGING VAULT_NAME)
 	$(call ASSERT_ANY_VAR_EXISTS, HEROKU_APP_EU HEROKU_APP_US)
-# Reset repository so that the app deploys on rebuilds even though there is no code change
+	# Reset repository so that the app deploys on rebuilds even though there is no code change
```

Comment:
minor style point, but if this comment is indented it will be interpreted as a command and passed to bash. which is fine, bash will see it as a comment as well, but it'll get printed to the console by make